packages/okta-angular/test/e2e/harness/e2e/page-objects/okta-signin.po.ts

```diff
@@ -18,8 +18,10 @@ export class OktaSignInPage {
       browser.waitForAngularEnabled(false);
     }
 
-    waitUntilVisible() {
-      browser.wait(ExpectedConditions.urlContains('/login'), 5000);
+    waitUntilVisible(issuer: string) {
+      // Check for a redirect to the Okta Hosted Login Page
```

Comment:
Do we need the whole authorize url to be constructed here?
Can't we just wait if url contains `/oauth2/default/v1/authorize`?

Reply:
When we redirect to Okta for login, we modify the current URL by extracting the `/authorize` query params into a stored reference (`okta_key`).

The only consistent URL is the `baseURL` of our org.

Reply:
In that case should we rename `authorizeUrl` to `baseUrl`?